packages/javascript/bh-shared-ui/src/hooks/useSearch/useSearch.tsx

```diff
@@ -52,7 +52,7 @@ export const useSearch = (keyword: string, type: ActiveDirectoryNodeKind | Azure
 };
 
 export const getKeywordAndTypeValues = (
-    inputValue: string
+    inputValue = ''
```

Comment:
This is the important part of the fix. Most everything else is noise from moving the search reducer to the shared ui

Reply:
Thank you for pointing this out! This looks like a pretty straightforward fix after filtering out the package change noise.